jobs/build-development.Jenkinsfile

```diff
@@ -6,7 +6,10 @@ node {
 }
 
 properties([
-    pipelineTriggers(pipeutils.get_push_trigger()),
+    pipelineTriggers([
+        // run every 24h only for now
+        cron("H H * * *")
```

Comment:
I imagine this timer wouldn't coincide/run at exactly the same time each day as `build-mechanical` right? Otherwise we'd want to do something different.

Reply:
Jenkins have a built-in feature to randomize the start hours while respecting the periodicity, when you write `H`
https://www.jenkins.io/doc/book/pipeline/syntax/#cron-syntax